blazor/datagrid/column-menu.md

```diff
@@ -70,6 +70,87 @@ The default menu items are displayed in the following table,
 The following image represents DataGrid with column menu property enabled,
 ![Blazor DataGrid with Column Menu](images/blazor-datagrid-column-menu.gif)
 
+## Custom column menu
+
+Custom column menu items can be added by defining [ColumnMenuItems](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.SfGrid-1.html#Syncfusion_Blazor_Grids_SfGrid_1_ColumnMenuItems) as a collection of [ColumnMenuItemModel](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.ColumnMenuItemModel.html) class. Actions for the customized column menu item can be defined in the [ColumnMenuItemClicked](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridEvents-1.html#Syncfusion_Blazor_Grids_GridEvents_1_ColumnMenuItemClicked) event of the Grid.
+
+In the following sample, we have changed the freeze direction of the column using the custom column menu:
```

Comment:
instead of changing the freeze direction,. First freeze one column initially then dynamically if i click the any column column menu item called Freeze then the corresponding column also need to freeze that is enough